jetty/content.md

```diff
@@ -26,10 +26,10 @@ The default Jetty environment in the image is:
 
 	JETTY_HOME    =  /usr/local/jetty
 	JETTY_BASE    =  /var/lib/jetty
-	JETTY_CONF    =  /usr/local/jetty/etc/jetty.conf
-	JETTY_STATE   =  /run/jetty/jetty.state
-	JETTY_ARGS    =
-	JAVA_OPTIONS  =
+	JETTY_CONF    =  /usr/local/jetty/etc/jetty.conf # Deprecated
```

Comment:
I think I'd probably just drop all these deprecated variables from the docs, unless we want to explain that they'll only work with `jetty.sh`, not with `start.jar`.